testpilot/frontend/static-src/styles/modules/_main-header.scss

```diff
@@ -7,6 +7,8 @@
     font-weight: 700;
     margin: 10px 0;
 
+    a { cursor: pointer; }
```

Comment:
@ktbee Cool, I wasn't sure if the pointer styles seemed off, or something. To update the PR with this line changed, you just need to update the commit, then force-push to the existing branch.

There are many ways to alter a git commit, but the simplest approach here is probably to check out this branch, remove that line from the stylesheet, then 'git add' the file, and 'git commit --amend --no-edit' to apply the new changes to the changes in the old commit. After doing this, git log should just show the one commit, without the style changes.

Once you've got the commit looking good locally, you can use 'git push -f' to the existing branch, and it'll overwrite the remote (github's copy of the branch) with the new commit.

Feel free to ping me with any questions. Git has a steep learning curve, but it's an amazing tool once you get it. The pro git book is a great learning resource: https://progit.org